### ISSUE TITLE
Time setup 

### ISSUE THREAD

**Issue by Sven9719** [InstaPy/InstaPy#6335]
Is it possible to set that after the daily follower will not continue until the next morning at 8 am?

**Comment by Sven9719** [comment]
Yes, but If I does a limit of 100 followers a day in Quota Supervisor it doesn’t stop it, it takes only a sleep to the next day and starts in the night again... And scheduling is only to start it or?

**Comment by Sven9719** [comment]
@elulcao Yes, but If I does a limit of 100 followers a day in Quota Supervisor it doesn’t stop it, it takes only a sleep to the next day and starts in the night again... And scheduling is only to start it or?

**Comment by elulcao** [comment]
> @elulcao Yes, but If I does a limit of 100 followers a day in Quota Supervisor it doesn’t stop it, it takes only a sleep to the next day and starts in the night again... And scheduling is only to start it or?

I suspect that's kind of risky, because the connection will be open all the day. That looks like a non-human action and more like an automated bot. 
The recommendation is to emulate a human behavior, closing and opening the app time to time.
